### ISSUE TITLE
Importing pypub

### ISSUE THREAD

**Issue by fmorato** [wcember/pypub#2]
Any support for python3 on the plans? =]

**Comment by wcember** [comment]
Hi Felipe-

Thanks for reaching out. I'm still developing pypub, but no plans right now for supporting python 3.

Curious how you found the package (or any feedback if you've tried to use it).

-Bill

**Comment by fmorato** [comment]
Hey Bill, 

Thanks for the quick reply.
It's on GitHub, and pipy. I've been searching for libraries that creates nice epubs. Your's seems to be one of the most complete ones (and the first one I found). But I haven't tried it as I'm using python3.
I tried the 2to3 but it can't properly do it and I don't really know what are all the coding differences.

I'm building a simple Flask app to convert Blog Articles to epubs and possibly converting to kindle and storing on a personal server (maybe with an automatic sync with my reader). I'm just learning programming and python, it goes quite slow.
